packages/puppeteer-core/src/common/bidi/utils.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2023 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {debug} from '../Debug.js';
+import * as Bidi from 'chromium-bidi/lib/cjs/protocol/protocol.js';
+import {Connection} from './Connection.js';
+
+/**
+ * @internal
+ */
+export const debugError = debug('puppeteer:error');
+/**
+ * @internal
+ */
+export async function releaseReference(
```

Comment:
it seems it would be suitable to be a method on the JSHandle instead of an util?

Reply:
We will want to release the ConsoleLogEntry as the arguments for it are RemoteValue as well. I we do something similar for CDP.